src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -318,7 +318,7 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
 
             setTxId(txId)
             setSerializedTxIndex(
-              serializeTxIndex(runeAccountId, txId, account, { memo, parser: 'swap' }),
+              serializeTxIndex(runeAccountId, txId, account, { memo, parser: 'thorchain' }),
```

Comment:
Will this serialize the Tx index in txHistory without calldata appended? If so, that will bork Tx detection

Reply:
This should keep behavior the same. I updated the parser type to `thorchain` so in changing this here. it will basically append the memo for unique identifier to distinguish synthetic transactions apart (IIRC).

Reply:
Confirmed the serialization is still the same ✅  get in!